modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -15,6 +15,7 @@
 
   before do
     stub_poa_verification
+    stub_mpi
```

Comment:
Put this in the top-level before and removed from all the individual tests to remove some repetitive code. Note there are some cases where we stub MPI not found to override this

Reply:
hmmm, im surprised this would work this way... typically an "allow_any_instance_of" run more than once will just stack responses, not override previous ones.

Reply:
maybe "override" was the wrong word choice. I can look into the exact reason why this doesn't cause issues and revert if explicitly mocking per test is easier to reason through